data/data/install.openshift.io_installconfigs.yaml

```diff
@@ -2286,10 +2286,39 @@ spec:
                       NetworkProjectID specifies which project the network and subnets
                       exist in when they are not in the main ProjectID.
                     type: string
+                  privateDNSZone:
+                    description: PrivateDNSZone Technology Preview. PrivateDNSZone
+                      contains the zone ID and project where the Private DNS zone
+                      will be created.
+                    properties:
+                      id:
+                        description: ID Technology Preview. ID or name of the zone.
+                        type: string
+                      project:
+                        description: ProjectID Technology Preview When the ProjectID
```

Comment:
Nit. Missing a period here Preview "." When.

Reply:
well its tech preview does it matter = )